src/forge/create.py

```diff
@@ -549,38 +537,36 @@ def get_placement_az(config, instance_details, mode=None):
     return az
 
 
-def create_fleet(n, config, task, instance_details):
+def create_fleet(n, config: Configuration, task, instance_details):
     """creates the AWS EC2 fleet
 
     Parameters
     ----------
     n : str
         Fleet name
-    config : dict
+    config : Configuration
         Forge configuration data
     task : str
         Forge service to run
     instance_details: dict
         EC2 instance details for create_fleet
     """
-    profile = config.get('aws_profile')
-    valid = config.get('valid_time', DEFAULT_ARG_VALS['valid_time'])
-    excluded_ec2s = config.get('excluded_ec2s', None)
-    tags = config.get('tags')
-    region = config.get('region')
+    valid = config.valid_time or DEFAULT_ARG_VALS['valid_time']
```

Comment:
Maybe for later but I wonder if we can bake these DEFAULT_ARG_VALS into the config object itself to simplify the code.

Reply:
I wanted to do that but I think we do access these defaults elsewhere and we can't directly access them from the Configuration object.